src/common/SPELLS_HUNTER.js

```diff
@@ -102,6 +102,28 @@ export default {
     name: 'Vulnerable',
     icon: 'ability_hunter_mastermarksman',
   },
+  
+  //CATEGORY
+  AUTO_SHOT_FOCUSMODULE: {
+	id: 75,
+	name: 'Auto Shot',
+	icon: 'ability_hunter_focusfire',
+  },
+  CRITICAL_FOCUS_FOCUSMODULE: {
+	id: 215107,
+	name: 'Critical Focus',
+	icon: 'ability_druid_replenish',
+  },
+  MULTISHOT_FOCUSMODULE: {
```

Comment:
What's the difference between this and http://www.wowhead.com/spell=2643/multi-shot ?

Reply:
Just the ID. This the id that corresponds with multi-shot with the events data. http://www.wowhead.com/spell=213363/multi-shot actually less correct (it's the lvl 1 version, whereas 2364 is the lvl 16-110 version), but that's the id I get. I originally translated it to 2643 within focus.js, but I decided it would probably be a better practice to simply leave the id as is and just provide the data for that id in SPELLS_HUNTER.js

Reply:
The naming for keys for spells should be the exact (English) spell name, all caps snake case. If a name is already used, both names should be changed to indicate what they are. E.g. often there's a SPELL_CAST and a SPELL_DAMAGE, SPELL_BUFF, SPELL_HEAL and/or SPELL_TRAIT. Rarely it's more complex than that like ATONEMENT_HEAL_NON_CRIT, ATONEMENT_HEAL_CRIT, ATONEMENT_BUFF. Blizzard usually doesn't create duplicate spells for no reason, so finding out why it's there and what it's for - and giving it a name that makes it clear - will usually help you reason about it and apply them properly.